src/basics/Links.js

```diff
@@ -155,7 +155,7 @@ export const Link = ({ href, newTab, ...props }) => {
       return <BasicLink href={url} {...props} />;
     case LINK_DESTINATIONS.docs:
     case LINK_DESTINATIONS.hash:
-      return <BasicLink href={url} {...props} />;
+      return <BasicLink onClick={() => navigate(url)} href={url} {...props} />;
```

Comment:
Hmmm I'm not sure we should wholesale override the click behavior for `BasicLink`. I see there's [a special case for hash links](https://github.com/stellar/new-docs/pull/169/files?diff=split&w=1#diff-94b0c1120e76c59fec00fd506741f52bR51), maybe something needs to change there?

Reply:
Gotcha! I thought `LINK_DESTINATIONS.hash` would be for hash links. I like adding it to `makeLinkedHeader`!

Reply:
I'm going to update Gatsby and see if it fixes

Reply:
Some updates - hash anchor tags work fine in react components. Just the ones that are addressed in .mdx files seem to be broken.